force-app/main/default/lwc/nksSubtitleLinker/nksSubtitleLinker.js

```diff
@@ -0,0 +1,56 @@
+import { LightningElement, wire, api } from 'lwc';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import showVideoTrackURL from '@salesforce/apex/NKS_VideoPlayerCtrl.showVideoTrackURL';
+import { isSubtitleFile } from 'c/utils';
+import SUBTITLE_LINK from '@salesforce/label/c.NKS_Subtitle_Link';
+import COPY_SUCCESS from '@salesforce/label/c.NKS_Copy_Message_Success';
+import COPY_FAIL from '@salesforce/label/c.NKS_Copy_Message_Fail';
+
+export default class NksSubtitleLinker extends LightningElement {
+    @api filetype;
+    @api recordId;
+
+    labels = {
+        SUBTITLE_LINK
+    };
+
+    get isSubtitleFile() {
+        return isSubtitleFile(this.filetype);
+    }
+
+    videoTrackURL;
+    @wire(showVideoTrackURL, { videoId: '$recordId'})
+    wiredShowVideoTrackURL(result) {
+        if (result.error) {
+            console.log(result.error);
+        } else if (result.data) {
+            this.videoTrackURL = result.data;
+        }
+    }
+
+    handleCopy() {
+        let copyValue = this.videoTrackURL;
+        let hiddenInput = document.createElement('input');
+        hiddenInput.value = copyValue;
+        document.body.appendChild(hiddenInput);
+        hiddenInput.focus();
+        hiddenInput.select();
+        try {
+            var successful = document.execCommand('copy');
+            this.showCopyToast(successful ? 'success' : 'error');
```

Comment:
Æ kom på en ting faktisk, vi ble bedt om å fjærne alle toasts for kopieringe (så lenge det ikke va en feil/error). Det har igjen litt med UU, men æ trur også den ka lite verdi te de seende veilederne også.